src/components/AvatarList/AvatarList.tsx

```diff
@@ -45,7 +50,7 @@ export const AvatarList: FunctionComponent<AvatarListProps> = ({ avatars, isOpen
 					<DropdownContent>
 						<Fragment>
 							{hiddenAvatars.map((avatar, index) => (
-								<a key={index} className="c-menu__item" href="#">
+								<a key={index} className="c-menu__item" href="#anchor">
```

Comment:
href links will reload the whole react app
it's better to work with onClick handler that returns the clicked avatar info.

```
onClick: (avatar: AvatarWithSubtitle) => void
```